### ISSUE TITLE
"ValueError: Cannot set flags on array scalars" when summing a single column FrameGO

### ISSUE THREAD

**Issue by ForeverWintr** [InvestmentSystems/static-frame#101]
It seems static_frame is internally summing to a single value, then trying to set flags on it. 

```python
f = sf.FrameGO(index=range(3))
f['a'] = 5
f.sum()
```
```
Traceback (most recent call last):
  Python Shell, prompt 27, line 1
    # Used internally for debug sandbox under external interpreter
  File "/home/rutherford/.env37/lib/python3.7/site-packages/static_frame/core/container.py", line 277, in func
    dtype=dtype)
  File "/home/rutherford/.env37/lib/python3.7/site-packages/static_frame/core/frame.py", line 2039, in _ufunc_axis_skipna
    dtype=dtype)
  File "/home/rutherford/.env37/lib/python3.7/site-packages/static_frame/core/type_blocks.py", line 720, in ufunc_axis_skipna
    result.flags.writeable = False
builtins.ValueError: Cannot set flags on array scalars.
```

**Comment by flexatone** [comment]
Thanks for posting this valuable find; I will get the fix in the next release.

**Comment by flexatone** [comment]
This issue is now fixed and will be available in the 0.4.1 release.

```python
In [2]: f = sf.FrameGO(index=range(3))                                                                                                                                                                                  

In [3]: f['a'] = 5                                                                                                                                                                                                      

In [4]: f.sum()                                                                                                                                                                                                         
Out[4]: 
<Series>
<Index>
a        15
<<U1>    <int64>
```
